corehq/apps/userreports/es/adapter.py

```diff
@@ -0,0 +1,44 @@
+from elasticsearch import NotFoundError
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.elastic import get_es_new
+
+
+class IndicatorESAdapter(IndicatorAdapter):
+
+    def __init__(self, config):
+        super(IndicatorESAdapter, self).__init__(config)
+        self.es = get_es_new()
+        self.table_name = get_table_name(config.domain, config.table_id)
+
+    def rebuild_table(self):
```

Comment:
ES has a fairly capable dynamic mapping on by default, it's on my list of things that should be looked at, but it's not strictly needed